roles/openshift_metrics/templates/route.j2

```diff
@@ -16,6 +16,10 @@ spec:
 {% if tls is defined %}
   tls:
     termination: {{ tls.termination }}
+{% if tls.ca_certificate is defined and tls.ca_certificate | length > 0%}
+    CACertificate: |
+{{ tls.ca_certificate|indent(6, true) }}
+{% endif %}
```

Comment:
The key and cert (and CA cert) correspond to what the user is passing in with the hawkular-metric.pem and hawkular-metrics-ca.cert, it has nothing to do with our generated certificates.

if a user passes this as a parameter, then its needed, if the user does not provide this, then its not required and it will use the router's default certificates